components/apimgt/org.wso2.carbon.apimgt.micro.gateway/org.wso2.carbon.apimgt.micro.gateway.configurator/src/test/resources/carbon-home/repository/conf/multitenancy/stratos.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?><!--
+  ~ Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
```

Comment:
Update here as well.

Reply:
These are not files that I have implemented, this is maintained as a resource for testing purposes. So, updating the license year is not required IMO.